site/source/index.md

```diff
@@ -54,33 +54,6 @@ Here are two Spacebars templates from an example app called "Leaderboard" which
 The template tags `{{name}}` and `{{score}}` refer to properties of the data context (the current player), while `players` and `selected` refer to helper functions.  Helper functions and event handlers are defined in JavaScript:
 
 
-```html
```

Comment:
Because the code example and that paragraph of text is shown twice (one right after the other). Take a look:
https://github.com/meteor/blaze/blob/master/site/source/index.md